huggingface/pytorch/training/docker/2.1/py3/sdk2.19.1/Dockerfile.neuronx

```diff
@@ -20,25 +21,31 @@ RUN pip install --no-cache-dir \
 	transformers[sklearn,sentencepiece,audio,vision]==${TRANSFORMERS_VERSION} \
 	datasets==${DATASETS_VERSION} \
 	optimum-neuron==${OPTIMUM_NEURON_VERSION} \
-    peft
+    peft \
+	gevent==${GEVENT_VERSION}
```

Comment:
gevent is a crucial dependency for inference server, please add this PR into the Q&A session for image patching, this is kinda of experience we want to document for future references. Thanks

Reply:
Nice, thanks for the context @haixiw , very similar discussion here https://github.com/aws/deep-learning-containers/pull/4365#issuecomment-2435408243